pkgs/applications/networking/cluster/terraform-providers/libvirt/default.nix

```
@@ -48,7 +40,9 @@ buildGoPackage rec {
 
   # Terraform allow checking the provider versions, but this breaks
   # if the versions are not provided via file paths.
-  postBuild = "mv go/bin/terraform-provider-libvirt{,_v${version}}";
+  postBuild = "mv $GOPATH/bin/terraform-provider-libvirt{,_v${version}}";
```

I had to also add the following changes for terraform to find this provider in the registry:

```suggestion
  postBuild = "mv $GOPATH/bin/terraform-provider-libvirt{,_v${version}}";
  
  ldflags = [ "-X main.version=${version}" ];
  passthru.provider-source-address = "registry.terraform.io/dmacvicar/libvirt";
```